apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx

```diff
@@ -207,7 +259,7 @@ export const ApplicationDetailsFooter = ({
             </Tooltip>
           )}
 
-          {isApplicationId(entity.id) && (
+          {isApplicationId(entity.id) && isMyApp && (
```

Comment:
this condition is incorrect for `Unpublish`
```suggestion
          {isApplicationId(entity.id) && (isMyApp || isPublicApp) && (
```